e2e/cypress/support/commands.js

```diff
@@ -84,11 +84,13 @@ Cypress.Commands.add("login", () => {
 });
 
 Cypress.Commands.add("consentAllowAll", () => {
-  cy.wait(1000);
-  cy.get("#CybotCookiebotDialog").then(($box) => {
-    if ($box.is(":visible")) {
-      cy.get("#CybotCookiebotDialogBodyLevelButtonLevelOptinAllowAll").click();
-    }
-  });
-  cy.wait(1000);
+  cy.get("#CybotCookiebotDialog")
+    .should("be.visible")
+    .then(($box) => {
+      if ($box.is(":visible")) {
+        cy.get(
+          "#CybotCookiebotDialogBodyLevelButtonLevelOptinAllowAll"
+        ).click();
+      }
+    });
```

Comment:
Vi må lige se om det her holder. Men der mangler i hvert fald nok nogle udvidede Timeouts

Reply:
nå, god ide, har tilføjet to. nu må vi se :-)